lib/src/git.dart

```diff
@@ -50,6 +50,37 @@ class GitException implements ApplicationException {
 /// Tests whether or not the git command-line app is available for use.
 bool get isInstalled => command != null;
 
+/// Splits the [output] of a git -z command at \0.
+///
+/// The first [skipPrefix] bytes of each substring will be ignored (useful for
+/// `git status -z`). If there are not enough bytes to skip, throws a
+/// [FormatException].
+List<Uint8List> splitZeroTerminated(Uint8List output, {int skipPrefix = 0}) {
```

Comment:
`splitZeroTerminated` without the `skipPrefix` might have been more sane.

But this is also fine :D

Reply:
Yeah - more sane - just I wanted to avoid creating too much garbage, and (as far as I can tell) `utf8.decode` cannot work on a partial buffer...